src/docker-compose.yml

```diff
@@ -3,33 +3,31 @@ x-transport: &transport-env
   env_file: "rabbit.env"
    
 services:
+  rabbitmq:
+    container_name: "rabbitmq"
+    image: "rabbitmq:management-alpine"
+    healthcheck:
+      test: rabbitmq-diagnostics -q ping
+      interval: 30s
+      timeout: 30s
+      retries: 3
   service-control-db:
     container_name: "servicecontrol-db"
     image: "particular/servicecontrol-ravendb:latest"
-    healthcheck:
```

Comment:
@ramonsmits  Because we depend on `service_healthy`, and that is already using the defined healthcheck in the image itself, see https://hub.docker.com/layers/particular/servicecontrol-ravendb/latest/images/sha256-de9d36535550e313c5cce94acd306b79e6519bef314ffbecb78c70352014747e?context=explore

![image](https://github.com/user-attachments/assets/6aad4c1f-8044-47fa-bc6e-8773ccbb20e5)

Reply:
So we don't need to create our own healthcheck